DIGNDB.App.SmitteStop.API/ContainerRegistration.cs

```diff
@@ -67,7 +66,7 @@ public static IServiceCollection AddAPIDependencies(this IServiceCollection serv
                 var xmlPath = Path.Combine(AppContext.BaseDirectory, xmlFile);
                 c.IncludeXmlComments(xmlPath);
             });
-            services.AddSingleton(new AuthOptions(environment.IsDevelopment()));
```

Comment:
Sure. But I recommend reducing differences between development and production environments. I didn't examine the details, but what's needed to get MobileAuthorization working on local development?

Reply:
I agree with reducing differences! 

`MobileAuthorization` needs this value to be set in `appsettings.json`:
![image](https://user-images.githubusercontent.com/9083085/102352012-9d88ab00-3fa7-11eb-96db-f44259e5aa41.png)

This value is a static secret shared between Mobile App and Backend.
Unfortunately it is encrypted using `Windows Data Protection API`, which makes it very hard to have 'a development value'.
It is hard because the encryption is being done using Windows Account credentials, so each user needs to have a different value there.

Because of this inconvenience we decided to disable the MobileAuthorization check on development environments.

Reply:
I think more research on the part of the dev team would be prudent here, but for now, I'll revert this change.